test/developerTest.js

```diff
@@ -0,0 +1,56 @@
+const assert = require('chai').assert;
+const devModel = require('../src/models/developer');
+const projectModel = require('../src/models/project');
+
+let dev = new devModel.Developer('Kristian', 'Free', 'pleb');
+let project = new projectModel.Project('Tonse Project', new Date(), new Date(2020, 4, 2), 50);
+let TTT = new projectModel.Project('Det lille project', new Date(), new Date(2020, 4, 11), 10);
```

Comment:
Jeg ville gerne have at de her ting kom ind i en beforeEach metode, så testene ikke brugte samme data :smile: [Documentation her](https://blog.logrocket.com/a-quick-and-complete-guide-to-mocha-testing-d0e0ea09f09d/)
```JS
let dev, project, TTT;

describe('developer', () => {
  beforeEach(function() {
    dev = new devModel....;
    project = new projectModel....;
    TTT = new .....;
  })
});
```